wordpress-mobile/WordPress-iOS
# Add share button to post preview

This PR adds a share button to the preview screen. Fixes #5425

### To test:

1. Go to My Sites > Select a site > Blog Posts
2. Tap on the "View" (published posts) or "Preview" (draft/scheduled posts) link at the bottom of a post card, e.g. https://cloudup.com/cbT75rpRYfD

You should now see a share button in the top right: https://cloudup.com/cn-B7Y3m6C0

__Needs review: @astralbodies__

Tried to find a way to remove duplicating code without a lot of refactoring. Hoping this is a decent compromise between the two.
